GETTING_STARTED.md

```diff
@@ -0,0 +1,602 @@
+
+## Setting up Test Kitchen
+
+Now let's set up Test Kitchen.  If you have used Test Kitchen before, setting it up with kitchen-terraform is a bit more complicated than other setups, but still doable!
+
+Go ahead and open your .kitchen.yml file using your favorite text editor
+
+```
+  $ vim .kitchen.yml
+```
+
+You'll see content in it that looks like this:
+
+kitchen.yml
+```
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_solo
+
+platforms:
+  - name: ubuntu-14.04
+  - name: centos-7.2
+
+suites:
+  - name: default
+    run_list:
+    attributes:
+```
+
+Let's go ahead and modify this for our needs, first the driver.  Rather than using the vagrant driver, we're going to use the terraform driver.
+
+Change these lines
+
+```
+---
+driver:
+  name: vagrant
+
+```
+
+To this:
+
+```
+---
+driver:
+  name: terraform
+
+```
+
+Now let's update the provisioner.  The provisioner we will use is also called terraform.
+
+Change this
+
+```
+provisioner:
+  name: chef_solo
+
+```
+
+To this:
+
+```
+provisioner:
+  name: terraform
+  variable_files:
+    - testing.tfvars
+```
+
+Notice that we are specifying what file will contain the values for the variables used by our terraform config.
+
+And let's update the platforms, for now we will only be testing on ubuntu.  So go ahead and change this:
+
+```
+platforms:
+  - name: ubuntu-14.04
+  - name: centos-7.2
+```
+
+To this:
+
+```
+platforms:
+  - name: ubuntu
+```
+
+So, right now, you .kitchen.yml should look like this:
+
+.kitchen.yml
+```
+---
+driver:
+  name: terraform
+
+provisioner:
+  name: terraform
+  variable_files:
+    - testing.tfvars
+
+platforms:
+  - name: ubuntu
+
+suites:
+  - name: default
+    run_list:
+    attributes:
+```
+
+Next, let's add a new section - the transport section.  This is what will allow kitchen-terraform to ssh into your terraform'd instances and run your inspec tests.  You will need to include the path to the private key half of your AWS keypair - wherever you keep it on your workstation.
+
+```
+transport:
+  ssh_key: ~/path/to/your/private/aws/key.pem
+```
+
+And now let's add another section, the verifier section.  The verifier is what will verify whether your test kitchen instances match what is laid out in your inspec files.
+
+```
+verifier:
+  name: terraform
+  format: doc
+  groups:
+  - name: inspec
+    tests:
+      -default
+    hostnames: public_dns
+    username: ubuntu
+```
+
+And let's stop and talk about what we did here - as this is where you will see some of the uniqueness of kitchen-terraform.
+
+With these lines:
+
+```
+verifier:
+  name: terraform
+  format: doc
+```
+
+We have specified that we are using the terraform verifier (which is included with kitchen terraform).  We are also using the format type of doc for our tests' output.
```

Comment:
"(which is included with kitchen terraform)" is awkward. There could be a comment further up that notes how the gem provides all 3 plugins.